### PR TITLE
Upgrade to o-grid v4

### PR DESCRIPTION
**Pull request by kaelig**
Also dimming the border color (was white) to make it a bit more subtle (cc @danskinner).

screenshot 2015-09-03 12 16 58

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,23 @@ Tested and working on:
...
+```diff
+ <nav class="o-footer__row o-footer__nav">
+ 	<div class="o-footer__col o-footer__col--full-width">
+ 		…
++ 		<div class="o-footer__divider"></div>
+ 	</div>
+ </nav>
+```
+
+## Upgrading from 2.x.x to 3.x.x
```
Comment (AlbertoElias) [inline README.md]: I think v4 shouldn't include the guide to v3

**Comment by AlbertoElias** [comment]
So why do we need a specific div for the divider?

File: README.md (hunk 1 above)
Comment (kaelig) [inline README.md]: That's a fair point

**Comment by kaelig** [comment]
> So why do we need a specific div for the divider?
Because of the way columns and rows now work (negative margins). I've tried with a pseudo selector instead but o-grid-row already has `:before` and `:after` set for the clearfix…

**Comment by AlbertoElias** [comment]
ok, lgtm

**Pull request merged**